doc/architecture-decisions/0017-moving-istio-configuration-out-of-this-repo.md

```diff
@@ -0,0 +1,44 @@
+# 17. Moving Istio and Related Configuration to CF-for-K8s Repo
+
+Date: 2020-09-15
+
+## Status
+
+Accepted
+
+## Context
+
+This ADR reverts the decision made in [ADR # 7. Maintain Generated
```

Comment:
Maybe insert the word `partially` here? We're still maintaining the generated Istio config it's just living in another place.

Also more generally this ADR should comment on our decision that we will be responsible for this config despite it being in a repo we don't directly control.